src/install-monerod-in-termux.sh

```diff
@@ -23,7 +23,17 @@ esac
 
 
 # Preconfigure
+if [ -d storage/downloads ]
```

Comment:
This line checks to see if you've already run termux-storage-setup and skips it if you have.

Seems to work as a workaround to the loss of storage access on certain devices.